docs/manual/java/guide/build/LagomBuild.md

```diff
@@ -68,7 +68,7 @@ We also recommend using Maven dependency management in your root project pom to
         </dependency>
         <dependency>
             <groupId>com.typesafe.play</groupId>
-            <artifactId>play-netty-server_2.11</artifactId>
+            <artifactId>play-akka-http-server_2.11</artifactId>
```

Comment:
Should we promote use of the `${scala.binary.version}` property (and default to 2.12)?

Reply:
I don't think we use `${scala.binary.version}` anywhere in the docs. 
When we support `2.12` in maven we'll have to update the docs and review these.

I don't think this is a change for this PR.